distribution/lib/Standard/Microsoft/0.0.0-dev/src/Internal/SQLServer_Dialect.enso

```diff
@@ -628,6 +628,7 @@ left = Base_Generator.lift_binary_op "LEFT" str-> n->
 right = Base_Generator.lift_binary_op "RIGHT" str-> n->
     SQL_Builder.code "right(" ++ str ++ ", CAST(" ++ n ++ " AS INT))"
 
+## PRIVATE
 _build_partition_sql grouping:SQL_Builder ordering:SQL_Builder -> SQL_Builder =
```

Comment:
Probably ok for consistency but tbh it shouldn't be needed because the whole Dialect file is `private` so it should never show up in CB regardless.

I think the `PRIVATE` annotations in dialects are a relic of times when we didn't yet have `private` modules. Now they could be removed. But nothing wrong with keeping them either I guess.